### ISSUE TITLE
Добавить поддержку метаданных

### ISSUE THREAD

**Issue by davitv** [marazmiki/django-selectel-storage#5]
Привет, Михаил! Мне нужно при выгрузке файлов добавлять метаданные (заголовки для Content-Disposition).
Предлагаю сделать так, как реализовано на амазоновской либе (см. [тут](https://github.com/etianen/django-s3-storage/blob/master/django_s3_storage/storage.py#L208)
Настройка AWS_S3_METADATA.
Как считаешь? Могу сделать пулл реквест

**Comment by marazmiki** [comment]
Здравствуйте!

Я так понимаю, что эти метаданные — некоторое отступление от "интерфейса" хранилища джанги?

**Comment by davitv** [comment]
Да. Но не совсем. Думаю, будет логично, что storage не только знает куда и как сохранить файл, но умеет при сохранении добавлять метаданные.

**Comment by marazmiki** [comment]
Может, это и логично, но уж коль скоро этой функциональности в других сторадж-бэкендов нет, это поведение нестандартное. Нужно это понимать.

С другой стороны, если они только в сохранении используются и выставляются неявно, внутри _save(), то можно и сделать, я не против :)

**Comment by davitv** [comment]
Согласен, поведение нестандартное. Но, если не знать об этой дополнительной возможности, она никак не повлияет на предсказуемость системы.
Ну так что, мне дописать и сделать пулл реквест?

**Comment by marazmiki** [comment]
А давайте :)

Только, пожалуйста, с тестами и кодом по PEP-0008, ок? Трэвис, если что, подскажет

**Comment by davitv** [comment]
Ок. Спасибо!
